functional-tests/bdd-tests/CRS_pay_licence_fee.feature

```diff
@@ -0,0 +1,14 @@
+﻿Feature: CRS_pay_licence_fee
+    As a logged in business user
+    I want to pay the Cannabis Retail Store fee
```

Comment:
Either "I want to pay the Cannabis Retail Store Licence Fee" or make abstract as "Pay_Licence_Fee"